### PR TITLE
Remove duplicate sponsor images from documentation

### PR DESCRIPTION
**Pull request by Zero3141**
## Description
Part of [#942](https://github.com/secureCodeBox/secureCodeBox/issues/942)

### Checklist

* [ ] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [ ] Make sure that all your commits are signed-off and that you are added to the [Contributors](https://github.com/secureCodeBox/secureCodeBox/blob/main/CONTRIBUTORS.md) file.
* [ ] Make sure that all CI finish successfully.
* [ ] Optional (but appreciated): Make sure that all commits are [Verified](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).

### CODE REVIEW

File: documentation/src/utils/config.js [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 const docsConfig = {
     repository: "secureCodeBox/secureCodeBox", // The repository url without the github part of the link
-    targetPath: "docs", // This needs to be 'docs' for the docusaurus build, but you may specify a 'docs/<subdirectory>'
+    targetPath: ".", // This needs to be '.' as files copied to docs (scanners & hooks) and static (sponsors) folder
```
Comment (Weltraumschaf) [inline documentation/src/utils/config.js:7]: Id do not really understand the comment here.

File: documentation/src/utils/config.js [hunk 2]
```diff
@@ -28,12 +28,16 @@ const docsConfig = {
...
-        dst: "scanners",
+        dst: "docs/scanners",
       },
       {
         src: "hooks",
-        dst: "hooks",
+        dst: "docs/hooks",
       },
+      {
+        src: "resources/sponsors",
```
Comment (Weltraumschaf) [inline documentation/src/utils/config.js:38]: Why that? We can simply reference the images IMO in the main README directly, right?

**Review by Weltraumschaf — CHANGES_REQUESTED** [review]
We should remove any unnecessary code to build things. So IMO the images should reside in the docusaurus folder, so we do not need to move them around. In the main README we can simply deep link to these files.

Also we should consider if we can simplify the scanner/hooks docs generation, to get rid ofthe untested build scripts.

**Comment by Zero3141** [comment]
Replaced by https://github.com/secureCodeBox/secureCodeBox/pull/1950

**Pull request closed without merging**